app/routes/test.js

```diff
@@ -2,14 +2,31 @@ const express = require('express');
 const router = express.Router();
 const issuesDB = require('../models/issue.store');
 
+var counter = 0;
```

Comment:
Use `let` instead than `var`

Reply:
Good catch! I have to upgrade my JS syntax to ES6. I'm from the old JS school 😄